packages/portal/tests/unit/server-middleware/api/keycloak.spec.js

```diff
@@ -0,0 +1,27 @@
+import nock from 'nock';
+
+import keycloak from '@/server-middleware/api/keycloak.js';
+
+const auth = { authorization: 'Basic Auth' };
+const config = {
+  origin: 'http://example.org',
+  realm: 'realm'
+};
+
+describe('server-middleware/api/keycloak', () => {
+  beforeAll(() => nock.disableNetConnect());
+  afterAll(() => nock.enableNetConnect());
+
+  describe('userId', () => {
+    it('extracts the ID from the user info', async() => {
+      keycloak.config = config;
+      nock(config.origin)
```

Comment:
Include the authorization header here with [nock.reqheaders](https://github.com/nock/nock/tree/v13.3.3?tab=readme-ov-file#specifying-headers)

Reply:
Yes! 
I had tried this, but kept getting errors with the requests not matching, so removed the headers to simplify it. It must have been something else and I never added it back in. Anyway, the headers are back in.